script/lib/util.py

```diff
@@ -27,10 +27,29 @@
 from .config import is_verbose_mode
 from .env_util import get_vs_env
 
+_PLATFORM_MAPPING = {
+    'cygwin': 'win',
+    'darwin': 'mac',
+    'linux2': 'linux',
+    'linux': 'linux',
+    'win32': 'win',
+}
+
 BOTO_DIR = os.path.abspath(os.path.join(__file__, '..', '..', '..', 'vendor',
                                         'boto'))
 
 
+def get_host_os():
+    """Returns the host OS with a predictable string."""
+    if sys.platform in _PLATFORM_MAPPING:
+        return _PLATFORM_MAPPING[sys.platform]
+
+    try:
+        return os.uname().sysname.lower()
```

Comment:
There's also `platform.system().lower() or sys.platform`

Do we need the fallback? `get_host_arch` is happy with what the `platform` library provides.

Reply:
everything we need for Brave should be covered with `_PLATFORM_MAPPING`.

If we failed there and got here, then most likely something needs to be fixed anyways, so any fallback here won't make things simpler.

This is basically a copy-paste from `gclient.py` https://chromium.googlesource.com/chromium/tools/depot_tools/+/refs/heads/main/gclient.py#1422

Reply:
I was wondering if the whole function could be a single `platform.system().lower()` evaluation for our purposes. Maybe we want to support mingw? Anyway, not important.